tests/TESTINS5.MLC

```diff
@@ -512,12 +589,29 @@ BIC_OK2  EQU    *
          RT1   CC2
          CL     R2,=F'6'
          RT1    CC0 
+         LM     R5,R6,=F'0,-3' #300 ADD 0 TESTS
+         SR      R0,R0
```

Comment:
Add "AHI   R0,1" to set CC not 0.

Reply:
You are right, I missed the last 0 result test change.  It's done now.